src/main/java/network/brightspots/rcv/TabulationCancelledException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Ranked Choice Voting Universal Tabulator
+ * Copyright (c) 2017-2019 Bright Spots Developers.
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+ * the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with this
+ * program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package network.brightspots.rcv;
+
+public class TabulationCancelledException extends Exception {
```

Comment:
For the other custom exceptions we've created, we've nested them within the most appropriate class. Maybe put this in `Tabulator` or `TieBreak`?

Reply:
Done, thanks.